controller/internal/enforcer/applicationproxy/markedconn/nativedata.go

```diff
@@ -0,0 +1,30 @@
+// +build !windows
+
+package markedconn
+
+import (
+	"net"
+)
+
+// NativeDataControl dummy impl
+type NativeDataControl struct {
+}
+
+// NewNativeDataControl returns initialized NativeDataControl
+func NewNativeDataControl() *NativeDataControl {
+	return &NativeDataControl{}
+}
+
```

Comment:
Not clear to me what "NativeData" means. Can we either document it or make the name more clear?

Reply:
Coming from the Windows world, I was using 'native' to mean "unmanaged" or outside the Go runtime/type system domain. I'll change it.

Reply:
Could be "UntypedData", "UnsafeData", "PlatformData". I'll try PlatformData unless there's another suggestion.